### PR TITLE
describe-schema: initial implementation

### LINKED ISSUE #327: `DESCRIBE SCHEMA AS SDL` shortcut
Now that we have the command `DESCRIBE SCHEMA AS SDL` [implemented](https://github.com/edgedb/edgedb/pull/2481), it would be nice to have a shortcut for it in REPL. Something like `\ds` perhaps.

### PR DESCRIPTION
**Pull request by tailhook**
Fixes #327

### CODE REVIEW

File: src/commands/backslash.rs [hunk 1]
```diff
@@ -30,6 +30,7 @@ const HELP: &str = r###"
 Introspection
   (options: -v = verbose, -s = show system objects, -I = case-sensitive match)
   \d [-v] NAME             describe schema object
+  \ds                      describe the whole schema
```
Comment (elprans) [inline src/commands/backslash.rs:33]: No long variant?

**Comment by 1st1** [comment]
Please don't merge this until we discuss how this should be called. Hopefully tomorrow

File: src/commands/backslash.rs (hunk 1 above)
Comment (tailhook) [reply src/commands/backslash.rs:33]: `\describe-schema`. Well, we list long variants here inconsistently :(

**Comment by tailhook** [comment]
Merged in. Will fix to the RFC1006 together with other commands.

**Pull request merged**